src/main/java/ua/nalezhytyi/hw19/task3/MongoDB.java

```diff
@@ -0,0 +1,44 @@
+package ua.nalezhytyi.hw19.task3;
+
+import com.mongodb.client.*;
+import com.mongodb.client.result.DeleteResult;
+import org.bson.BSON;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+public class MongoDB {
+    public static void main(String[] args) {
+
+        MongoClient mongoClient = MongoClients.create("mongodb://admin:admin123@localhost:27017");
+        MongoDatabase database = mongoClient.getDatabase("test");
+        MongoCollection<Document> collection = database.getCollection("humans");
+        Document doc = new Document();
+        //create human
+        doc.append("name", "Zoya")
+                .append("age", 21)
+                .append("city", "London")
+                .append("profession", "Manager")
+                .append("Sex", "female");
+        //collection.insertOne(doc);
+
+        //read human
+        FindIterable<Document> documents = collection.find();
+        MongoCursor<Document> cursor = documents.cursor();
+        while (cursor.hasNext()) {
+            Document next = cursor.next();
+            System.out.println(next);
+        }
```

Comment:
Since FindIterable alread is a Iterable object, we just can use for-loop with it, without getting a cursor (because it's a more low level abstraction)